platform/linuxbsd/wayland/display_server_wayland.cpp

```diff
@@ -643,6 +648,40 @@ bool DisplayServerWayland::screen_is_kept_on() const {
 #endif
 }
 
+bool DisplayServerWayland::screen_is_hdr_supported(int p_screen) const {
+	// If the compositor supports HDR then assume it can transform HDR for all screens
+	return wayland_thread.supports_hdr();
```

Comment:
What about multiple displays? How would you check that the display your window is a part of supports HDR?

We should be careful here and in the other `screen_` methods in assuming the main window is the only window that matters. Any window created by Godot could opt into HDR output and needs to know if the screen it is on supports HDR and the capabilities of that display.

Reply:
You shouldn't check displays at all, that's for the compositor to deal with. If it tells you to do "HDR" and can't handle that, that's not your problem to deal with.